runescape-client/src/main/java/class224.java

```diff
@@ -26,7 +29,8 @@
 		descriptor = "(II)V",
 		garbageValue = "2091444199"
 	)
-	static final void method4232(int var0) {
+	@Export("redrawMiniMapIcons")
```

Comment:
I think this does more than just draw icons - if I'm correct, it draws the entire minimap :P

Maybe `createMinimapSprite`?